src/client/webpack.config.js

```diff
@@ -25,6 +25,10 @@ module.exports = {
         alias: {
             '@': path.resolve(__dirname, '../..'),
             '@client': path.resolve(__dirname, '../../src/client'),
+            '@components': path.resolve(
+                __dirname,
+                '../../src/client/components'
```

Comment:
You know, I think this can be simpler/relative to where the webpack config file is. I know we did it this way for the `@client` alias, but I found that
```
@components': path.resolve(
                __dirname,
                './components'
            ),
```
works just as well. Might be worth making that relative path shorter for `@client` and `@components` in this PR.